services/path.py

```diff
@@ -121,25 +121,19 @@ def is_subpath(full_path: str, subpath: str):
     return full_path.startswith(f"{subpath}/") or full_path == subpath
 
 
-# Regex that separates string into 2 groups, taken from https://stackoverflow.com/a/33021907
-def calculate_path_file_and_dir_groups(path: str) -> tuple[str, str]:
-    result = re.search(r"((?:[^/]*/)*)(.*)", path)
-    return result.groups()
-
-
-def is_file(path) -> bool:
```

Comment:
@adrian-codecov I'm not sure this logic is correct.  Couldn't you have a file w/o a `.` in it?  Or a file with a leading `.`?

I think the only way to differentiate file vs. directory path is by checking the report.